doc/sphinxman/source/build_planning.rst

```diff
@@ -218,19 +218,24 @@ build system will automatically download and build.
 
 * QCElemental |w---w| `[what is this?] <https://qcelemental.readthedocs.io/en/latest/>`_
 
-* QCEngine |w---w| `[what is this?] <https://qcengine.readthedocs.io/en/latest/>`_
+* QCEngine |w---w| `[what is this?] <https://qcengine.readthedocs.io/en/latest/>`_ (March 2019; added by v1.4)
 
 Additionally, there are runtime-only dependencies:
 
 * NumPy http://www.numpy.org/
 
-* networkx https://github.com/networkx/networkx
+* networkx https://github.com/networkx/networkx (transitive dependency of QCElemental)
 
-* deepdiff https://github.com/seperman/deepdiff
+* pint https://pint.readthedocs.io/en/latest/ (transitive dependency of QCElemental)
 
-* pint https://pint.readthedocs.io/en/latest/
+* pydantic https://pydantic-docs.helpmanual.io/# (transitive dependency of QCElemental)
```

Comment:
`pint` and `pedantic` are required. Only networkx is not.

Reply:
Going to merge this and pin for a TODO as its not too crucial.

Reply:
true, but psi uses capabilites that demand it. I'll clarify.